app/uk/gov/hmrc/fhregistrationfrontend/views/business_type.scala.html

```diff
@@ -1,92 +1,112 @@
 @*
- * Copyright 2022 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *@
+* Copyright 2022 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
 
 @import uk.gov.hmrc.fhregistrationfrontend.forms.models.BusinessType.BusinessType
-@import uk.gov.hmrc.fhregistrationfrontend.views.helpers._
-@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers._
-@import uk.gov.hmrc.fhregistrationfrontend.forms.navigation.Navigation
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
 
 @this(
-        mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
+        layout: layout, viewHelpers: ViewHelpers, govukErrorMessage: GovukErrorMessage
 )
 
-@(businessTypeForm: Form[BusinessType], backUrl: String)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@(businessTypeForm: Form[BusinessType], backUrl: String)(implicit request: Request[_], messages: Messages,
+        appConfig: AppConfig)
+
+@pageScripts = {<script type="text/javascript" src='@routes.Assets.at("javascripts/fh.js")'></script>}
```

Comment:
do we need this page script? i thought we are removing it. If needed I will add it in my Prs as well

Reply:
Because we can't see what the script does, it is hard to know if we need them or not.  I know on EI we removed a script and it slightly changed how the service worked and we ended up adding it back at the end or after the service went live.     Are you aware of any way for us to verify what the specific scripts reference are?

Reply:
I don't know what it does, and it was not making any change when I removed it.
@Chrissystk do you have any idea what it does?